app/views/layouts/application.html.erb

```diff
@@ -55,11 +55,22 @@
             </span>
           </a>
         </div>
-          <div class="govuk-header__content">
-
+        <div class="govuk-header__content">
           <a href="#" class="govuk-header__link govuk-header__link--service-name">
```

Comment:
Should this link go somewhere?

Reply:
Not for the moment, but I'm going to add some reporting on the root path so I'll update this link as part of that change.